<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="similarityPercentageLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>570</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLabel" name="previousScreenshotLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="currentScreenshotLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="buttonFrame">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>600</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>141</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
